### ISSUE TITLE
Permit adapting plotting configuration to different drive types

### ISSUE THREAD

**Issue by fdachille** [ericaltendorf/plotman#22]
SS and HD drives require different plotting parameters for optimal speed. When there is a mixture of drives in a system, there should be a way to tailor the plotting parameters for each drive type, rather than just choose one.

This could be achieved by specifying a plotting configuration for each tmp drive, and then listing multiple plotting configurations, each with a name. Then, when initiating a plot, select the named plotting configuration for the tmp drive.

**Comment by ericaltendorf** [comment]
Agreed this should be done.  No ETA for implementation, but thanks for filing.
